appveyor.yml

```diff
@@ -27,43 +43,81 @@ install:
+  - ps: Get-Content ..\groonga\base_version | ForEach-Object { $env:GROONGA_VERSION = $_ }
+  - ps: Get-Content ..\groonga\include\groonga\version.h.in | ForEach-Object { $_ -replace "@GRN_VERSION@","$env:GROONGA_VERSION" } | Set-Content ..\groonga\include\groonga\version.h
   - rmdir /S /Q ..\groonga\test\
   - cd ..\groonga\vendor
-  - c:\Ruby22-x64\bin\ruby -v download_lz4.rb
-  - c:\Ruby22-x64\bin\ruby -v download_mecab.rb
+  - c:\Ruby26-x64\bin\ruby -v download_lz4.rb
+  - c:\Ruby26-x64\bin\ruby -v download_mecab.rb
   - cd ..\..\mariadb-%MARIADB_VERSION%
   - mkdir storage\mroonga\vendor
   - move ..\groonga storage\mroonga\vendor\groonga
   - git clone --quiet --depth 1 https://github.com/groonga/groonga-normalizer-mysql.git storage\mroonga\vendor\groonga\vendor\plugins\groonga-normalizer-mysql
-build_script:
-  - "echo # > win\\packaging\\CMakeLists.txt"
-  - cmake . -G "%CMAKE_GENERATOR_NAME%"
-      -DCMAKE_BUILD_TYPE=Debug
-      -DPLUGIN_ARCHIVE=NO
-      -DPLUGIN_BLACKHOLE=NO
-      -DPLUGIN_CASSANDRA=NO
-      -DPLUGIN_CONNECT=NO
-      -DPLUGIN_CSV=NO
-      -DPLUGIN_EXAMPLE=NO
-      -DPLUGIN_FEDERATED=NO
-      -DPLUGIN_FEDERATEDX=NO
-      -DPLUGIN_HEAP=NO
-      -DPLUGIN_INNOBASE=YES
-      -DPLUGIN_MYISAM=NO
-      -DPLUGIN_MYISAMMRG=NO
-      -DPLUGIN_OQGRAPH=NO
-      -DPLUGIN_PERFSCHEMA=NO
-      -DPLUGIN_SEQUENCE=NO
-      -DPLUGIN_SPHINX=NO
-      -DPLUGIN_SPIDER=NO
-      -DPLUGIN_TEST_SQL_DISCOVERY=NO
-      -DPLUGIN_TOKUDB=NO
-      -DPLUGIN_XTRADB=NO
-      -DWITH_UNIT_TESTS=OFF
-      -DWITH_MARIABACKUP=OFF
-      -DGRN_WITH_BUNDLED_MECAB=ON
-  - cmake --build . --config Debug
-
 test: off
+
+for:
+  -
+    matrix:
+      only:
+        - configuration: BuildCheck
+    build_script:
+      - "echo # > win\\packaging\\CMakeLists.txt"
+      - cmake . -G "%CMAKE_GENERATOR_NAME%"
+          -DCMAKE_BUILD_TYPE=RelWithDebInfo
+          -DPLUGIN_ARCHIVE=NO
+          -DPLUGIN_BLACKHOLE=NO
+          -DPLUGIN_CASSANDRA=NO
+          -DPLUGIN_CONNECT=NO
+          -DPLUGIN_CSV=NO
+          -DPLUGIN_EXAMPLE=NO
+          -DPLUGIN_FEDERATED=NO
+          -DPLUGIN_FEDERATEDX=NO
+          -DPLUGIN_HEAP=NO
+          -DPLUGIN_INNOBASE=YES
+          -DPLUGIN_MYISAM=NO
+          -DPLUGIN_MYISAMMRG=NO
+          -DPLUGIN_OQGRAPH=NO
+          -DPLUGIN_PERFSCHEMA=NO
+          -DPLUGIN_SEQUENCE=NO
+          -DPLUGIN_SPHINX=NO
+          -DPLUGIN_SPIDER=NO
+          -DPLUGIN_TEST_SQL_DISCOVERY=NO
+          -DPLUGIN_TOKUDB=NO
+          -DPLUGIN_XTRADB=NO
+          -DWITH_UNIT_TESTS=OFF
+          -DWITH_MARIABACKUP=OFF
+          -DGRN_WITH_BUNDLED_MECAB=ON
+      - cmake --build . --config RelWithDebInfo
+  -
+    matrix:
+      only:
+        - configuration: ReleasePackage
+    build_script:
+      - set
+      - cd %APPVEYOR_BUILD_FOLDER%\..
+      - git clone --depth 1
+        https://github.com/cosmo0920/PowerShell-for-Mroonga-building.git mrnbuild
```

Comment:
This repository is needless when we build our official package by AppVeyor.
Can we move all related staffs to this repository?

Reply:
I've imported related stuff.